crates/oxc_span/src/source_type.rs

```diff
@@ -71,7 +71,8 @@ impl Default for SourceType {
 }
 
 /// Valid file extensions
-pub const VALID_EXTENSIONS: [&str; 8] = ["js", "mjs", "cjs", "jsx", "ts", "mts", "cts", "tsx"];
+pub const VALID_EXTENSIONS: [&str; 9] =
+    ["js", "mjs", "cjs", "jsx", "ts", "mts", "cts", "tsx", "snap"];
```

Comment:
If it was removed, the test would fail as well, or oxc will not check the `.snap` type? or is there some other method or plugin can bypass this check?

Reply:
> Is there some other method or plugin can bypass this check?

Currently, no. We can comment out these failed tests.

Reply:
Done